reconcile/test/test_utils_terraform_client.py

```diff
@@ -0,0 +1,87 @@
+from unittest import TestCase
+import reconcile.utils.terraform_client as tfclient
+
+
+class TestDeletionApproved(TestCase):
+
+    def test_no_deletion_approvals(self):
+        account = {
+            'name': 'a1',
+            'deletionApprovals': []
+        }
+        tf = tfclient.TerraformClient(
+            'integ',
+            'v1',
+            'integ_pfx',
+            [account],
+            {},
+            1
+        )
+        result = tf.deletion_approved('a1', 't1', 'n1')
+        self.assertFalse(result)
+
+    def test_deletion_not_approved(self):
+        account = {
+            'name': 'a1',
+            'deletionApprovals': [
+                {
+                    'type': 't1',
+                    'name': 'n1',
+                    'expiration': '2000-01-01'
+                }
+            ]
+        }
+        tf = tfclient.TerraformClient(
+            'integ',
+            'v1',
+            'integ_pfx',
+            [account],
+            {},
+            1
+        )
+        result = tf.deletion_approved('a1', 't2', 'n2')
+        self.assertFalse(result)
+
+    def test_deletion_approved_expired(self):
+        account = {
+            'name': 'a1',
+            'deletionApprovals': [
+                {
+                    'type': 't1',
+                    'name': 'n1',
+                    'expiration': '2000-01-01'
+                }
+            ]
+        }
+        tf = tfclient.TerraformClient(
+            'integ',
+            'v1',
+            'integ_pfx',
+            [account],
+            {},
+            1
+        )
+        result = tf.deletion_approved('a1', 't1', 'n1')
+        self.assertFalse(result)
+
+    def test_deletion_approved(self):
+        account = {
+            'name': 'a1',
+            'deletionApprovals': [
+                {
+                    'type': 't1',
+                    'name': 'n1',
+                    'expiration': '2500-01-01'
```

Comment:
in 2500 people are going to freak out because of the economy crashing effect of "The Maor Bug"

Reply:
i figured its easier to use the year 2500 and not patch datetime :)